### DISCUSSION TITLE
account suspended

### DISCUSSION THREAD

**Discussion by Deepakkumar1322** [community/community#146141] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am writing to request assistance with accessing my suspended account. I am unsure of the reason for the suspension, as I did not receive any notice or explanation. I submitted a support request, and my ticket (#2937811) was last updated on September 20, 2024.

Since I have not received any further updates for over two months, I would greatly appreciate it if you could provide the current status of my case or let me know if any additional information is needed to resolve the issue.

**Comment by shinybrightstar** [comment]
Hi @Deepakkumar1322 

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.
